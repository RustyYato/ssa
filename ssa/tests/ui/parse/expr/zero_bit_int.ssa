let x : i0;
let x : u0;
