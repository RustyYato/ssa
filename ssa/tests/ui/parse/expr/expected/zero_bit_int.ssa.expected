zero bits signed integer [
    Token {
        kind: Ident(
            "i0",
        ),
        lexeme: "i0",
        line_start: 0,
        col_start: 8,
        line_end: 0,
        col_end: 10,
    },
    Token {
        kind: Semicolon,
        lexeme: ";",
        line_start: 0,
        col_start: 10,
        line_end: 0,
        col_end: 11,
    },
]
zero bits unsigned integer [
    Token {
        kind: Ident(
            "u0",
        ),
        lexeme: "u0",
        line_start: 1,
        col_start: 8,
        line_end: 1,
        col_end: 10,
    },
    Token {
        kind: Semicolon,
        lexeme: ";",
        line_start: 1,
        col_start: 10,
        line_end: 1,
        col_end: 11,
    },
]